gryphon/lib/exchange/binance_je.py

```diff
@@ -0,0 +1,188 @@
+"""
+https://github.com/binance-jersey/binance-official-api-docs
+"""
+
+# -*- coding: utf-8 -*-
+import hashlib
+import hmac
+import time
+
+from gryphon.lib.exchange import exceptions
+from gryphon.lib.exchange.exchange_api_wrapper import ExchangeAPIWrapper
+from gryphon.lib.logger import get_logger
+from gryphon.lib.models.exchange import Balance
+from gryphon.lib.money import Money
+
+logger = get_logger(__name__)
+
+
+def binance_signature(params, secret):
```

Comment:
Could this be a classmethod on the wrapper?

Reply:
It could. What's the benefit?

Reply:
Standardization, I don't think any other integrations leave their auth method as a global. I can imagine scenarios where you'd want to call this without instantiating a wrapper object though, that's why I suggested making it a classmethod.

Reply:
Fine. No great objection, just wondered!

Reply:
I went for a static method instead, given that it needs no access to any of the class members.